contrib/quince/plugins.yml

```diff
@@ -31,3 +31,37 @@
   maintainer: Emad Rad <codewithemad@gmail.com>
   description: |
     Effortlessly integrate Open edX into WordPress with this plugin, designed for smooth setup and reliable performance.
+
+- name: branding
+  src: tutor-contrib-branding>=17.0.0,<18.0.0
+  url: https://github.com/aulasneo/tutor-contrib-branding
+  author: Andrés González <andres@aulasneo.com>
+  maintainer: Andrés González <andres@aulasneo.com>
+  description: |
+    Easily change the look and feel of Open edX installations.
+    Change colors, logos, add custom CSS or change completely the appearance of your Open edX site.
+    With this plugin you will be able to control the appearance of both legacy pages based on HTML
+    (like the home page, course catalog, dashboard, etc.), as well as MFE components.
+
+- name: nudge
+  src: tutor-contrib-nudge>=17.0.0,<18.0.0
+  url: https://github.com/aulasneo/tutor-contrib-nudge
+  author: Andrés González <andres@aulasneo.com>
+  maintainer: Andrés González <andres@aulasneo.com>
+  description: |
+    Enable nudge and highlights emails.
+
+- name: panorama
+  src: tutor-contrib-panorama>=17.0.0,<18.0.0
+  url: https://github.com/aulasneo/tutor-contrib-panorama
+  author: Andrés González <andres@aulasneo.com>
+  maintainer: Andrés González <andres@aulasneo.com>
+  description: |
+    The ultimate analytics solution for Open edX and more.
+    Panorama is the most powerful and flexible tool to get deep learning insights from Open edX.
+    It allows connecting multiple Open edX sites in one single view so you can compare all your
+    LMS installations in one sight. It can also connect to other data sources, to enable advanced
+    analyses correlating data from your LMSs as well as your SIS, ERP, CRM or any other support systems.
+    Panorama requires several AWS services to store, process and represent data.
+    Connect Panorama for free to your AWS datalake, or visit https://aulasneo.com/panorama to know
```

Comment:
The description is markdown-formatted, so feel free to insert links with `[text](https://myurl.com)`. Also, be careful with line breaks: single line breaks will _not_ insert a new line.